source/fresh-installation.rst

```diff
@@ -344,8 +344,31 @@ Installing on a Personal Computer
 ---------------------------------
 
 You can install these packages on your personal laptop outside the controls
-network. Install miniconda or Ananconda, and create user environment as
+network. Install miniconda or Anaconda, and create user environments as
 described above. All of the packages are mirrored on anaconda.org, outside of
 the NSLS-II firewall, where you will be able to access them. The channels are
 called ``lightsource2-tag`` and ``lightsource2-dev`` instead of ``nsls2-tag``
-and ``nsls2-dev`` respectively.
+and ``nsls2-dev`` respectively. The following serves as a step by step guide:
+
+1.  Follow the instructions in the link below to install minconda or anaconda.
```

Comment:
Use `#.` instead of `1.` and let sphinx handle the numbering. I didn't used to do this, so you'll find counterexamples in our own docs, but I've become convinced we should switch going forward.